pkg/gittool/diff.go

```diff
@@ -0,0 +1,44 @@
+package gittool
+
+// Operation defines the operation of a diff item.
+type Operation int
+
+const (
+	// Equal item represents a equals diff.
+	Equal Operation = iota
```

Comment:
looks like there're a few other status from the git diff, so you simplified that only support partial of the opreation?


doc reference: https://git-scm.com/docs/git-diff
Possible status letters are:

A: addition of a file

C: copy of a file into a new one

D: deletion of a file

M: modification of the contents or mode of a file

R: renaming of a file

T: change in the type of the file (regular file, symbolic link or submodule)

U: file is unmerged (you must complete the merge before it can be committed)

X: "unknown" change type (most probably a bug, please report it)

Reply:
Should admit that that's would not be a complete list, but add and delete are the least supported operations.

Reply:
we need quite a few testing on the scope we supported.